src/WebJobs.Script/Binding/ServiceBusScriptBindingProvider.cs

```diff
@@ -104,10 +106,15 @@ public override Collection<Attribute> GetAttributes()
                 Collection<Attribute> attributes = new Collection<Attribute>();
 
                 string eventHubName = Context.GetMetadataValue<string>("path");
+                if (!string.IsNullOrEmpty(eventHubName))
+                {
+                    eventHubName = _nameResolver.ResolveWholeString(eventHubName);
```

Comment:
Here and below, we have to resolve, since this extension is doing early registration of EventProcessorHosts below via AddEventProcessorHost. That contrasts with regular attribute properties, where we don't have to resolve because they'll be resolved later.